docs/DeveloperGuide.md

```diff
@@ -2,24 +2,138 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Add Function
 
+#### System Architecture of the Add Function<br/>
+
+![](team/Architect-digram.PNG)
```

Comment:
For this diagram it looks like the user can by pass the interface and directly add commands? Its a bit confusing whether this was intended or why it was designed this way. Further explanation on this part of the diagram would be helpful

![image](https://user-images.githubusercontent.com/53657436/97387037-2708ee80-1910-11eb-9a5a-c944017f6abf.png)